scene/gui/scroll_bar.cpp

```diff
@@ -449,17 +449,14 @@ double ScrollBar::get_area_size() const {
 		area -= get_grabber_min_size();
 		return area;
 
-	} else if (orientation == HORIZONTAL) {
+	} else { //Horizontal
```

Comment:
Same issue as previously, if the enum changes, this catch all becomes a bug. Explicit is better, even if redundant.